css/styles.css

```diff
@@ -38,7 +38,7 @@ body {
 /* Global */
 .container {
   max-width: 120rem;
-  width: 88%;
+  width: 90%;
   margin: 0 auto;
 }
```

Comment:
- [ ] Kindly notice the background of the popup windows overlay, It have some blur design which is not implemented in you design. Kindly make sure you stick to the Figma design as much as possible.
### Your design :point_down: 
![Screenshot from 2023-05-17 17-12-44](https://github.com/lRebornsl/portfolio/assets/91142432/7b78d7b1-20ec-4d08-b314-daa4a4f1c54f)
### What it is supposed to look like :point_down: 
![Screenshot from 2023-05-17 17-39-10](https://github.com/lRebornsl/portfolio/assets/91142432/5476a3b1-f683-4c92-8be2-87304b8338bc)